joinmarket/support.py

```diff
@@ -290,7 +293,7 @@ def choose_sweep_orders(db,
                         txfee,
                         n,
                         chooseOrdersBy,
-                        ignored_makers=None):
```

Comment:
Comment also applies to L305-307.

See: https://github.com/JoinMarket-Org/joinmarket/blob/dc8cfd06aa7b0441ba5c903d814c28e5fde2ed34/joinmarket/__init__.py#L716-L729

@ghtdak explained to me that this change is due to a 'feature' in Python; see: http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments

I suggest reverting these changes.